### ISSUE TITLE
The install of flaregm.installer.vm was NOT successful

### ISSUE THREAD

**Issue by johnZhang17** [mandiant/VM-Packages#437]
### What's the problem?

Error while running 'C:\ProgramData\chocolatey\lib\flarevm.installer.vm\tools\chocolateyinstall.ps1'.

See log for details (C:\ProgramData\chocolatey\logs\[partial-chocolatey.log])
[partial-chocolatey.log](https://github.com/mandiant/VM-Packages/files/11760116/partial-chocolatey.log)

The error encountered is described in the first few lines of the chocolatey.log:
```
2023-06-15 17:36:27,257 6284 [DEBUG] - Running 'ChocolateyScriptRunner' for flarevm.installer.vm v0.0.0.20230606 with packageScript 'C:\ProgramData\chocolatey\lib\flarevm.installer.vm\tools\chocolateyinstall.ps1', packageFolder:'C:\ProgramData\chocolatey\lib\flarevm.installer.vm', installArguments: '', packageParameters: '', preRunHookScripts: '', postRunHookScripts: '',
2023-06-15 17:36:27,257 6284 [DEBUG] - Running package script 'C:\ProgramData\chocolatey\lib\flarevm.installer.vm\tools\chocolateyinstall.ps1'
2023-06-15 17:36:27,569 6284 [ERROR] - ERROR: Cannot validate argument on parameter 'ArgumentList'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
```
I searched on chocolateyinstall.ps1 the parameter "ArgumentList", which should not have a null argument, but did not find it.

Really don't know where to put my fingers on. Any help? :)

### Steps to Reproduce

1. Installed Virtual Box and created a new VM with Windows 10 Pro (build 19045)

2. Followed the instruction indicated (https://github.com/mandiant/flare-vm#installation) to prepare the VM

3. Opened a new instance of Powershell as Admin and launched the following commands in the following order:
  a) Set-ExecutionPolicy Unrestricted
  b) Unblock-File .\install.ps1
  c) .install.ps1 -password _'password'_ -noWait -noGui -noChecks -noReboots

Did not get any error, but the one for which I am opening this issue.

### Environment

Attached the output of **VM-Get-Host-Info**.
[VM-Get-Host-Info.txt](https://github.com/mandiant/VM-Packages/files/11760200/VM-Get-Host-Info.txt)

### Detected by test suite

I don't know

**Comment by mr-tz** [comment]
Looks like a duplicate of https://github.com/mandiant/flare-vm/issues/461. Can you please confirm you're using the most recent code and reopen if needed?
